pandas/core/generic.py

```diff
@@ -6076,8 +6078,34 @@ def asof(self, where, subset=None):
         --------
         merge_asof
 
-        """
+        Examples
+        --------
+        Take all columns into consideration
+
+        >>> data = u'''dt,a,b
```

Comment:
Probably best to construct this directly, rather than using read_csv. Can you do 

```python
df = pd.DataFrame({'a': [...], 'b': [...], index=pd.DatetimeIndex([...])
```

instead? Also, these floating point values are a bit hard to understand when just glancing. Could you change to integers instead?

Reply:
Done